src/js/fc.js

```diff
@@ -501,5 +501,16 @@ var FC = {
 
     getMcuType: function () {
         return FC.MCU_TYPES[CONFIG.mcuTypeId];
-    }
+    },
+
+    boardHasVcp() {
+        var hasVcp = false;
+        if (semver.gte(CONFIG.apiVersion, "1.37.0")) {
+            hasVcp = (CONFIG.commCapabilities & 0x01) !== 0;
```

Comment:
It will be better if we have the 0x01 defined as constant, I don't know where to place it in a clean way...